core/templates/dev/head/domain/improvements/PlaythroughImprovementCardObjectFactorySpec.ts

```diff
@@ -16,7 +16,6 @@
  * @fileoverview Unit tests for the PlaythroughImprovementCardObjectFactory.
  */
 
-require('domain/statistics/PlaythroughImprovementCardObjectFactory.ts');
```

Comment:
ditto.

Reply:
Same, resolving this so we have just one comment thread to talk on.